src/cocotb/simulator.pyi

```diff
@@ -50,6 +50,7 @@ class gpi_sim_hdl:
     def get_definition_name(self) -> str: ...
     def get_handle_by_index(self, index: int) -> gpi_sim_hdl | None: ...
     def get_handle_by_name(self, name: str) -> gpi_sim_hdl | None: ...
+    def get_handle_by_name_native(self, name: str) -> gpi_sim_hdl | None: ...
```

Comment:
I don't think this is needed anymore.

Reply:
correct, and the signature needed updating.